### ISSUE TITLE
Jabba the Hutt is a Cat with 98% certainty

### ISSUE THREAD

**Issue by sgerhardt** [fastai/fastbook#294]
While working through the first model to determine if a photo is a cat, I thought it'd be interesting to see how Jabba the Hutt faired. It resulted in "Yes" for the `is_cat` function [here](https://github.com/fastai/fastbook/blame/master/01_intro.ipynb#L574). with a 98% certainty. 

![JABBA](https://user-images.githubusercontent.com/4251684/95803183-124c1a00-0cc5-11eb-98a6-43660df16377.jpg)

I'm just dipping my toe into machine learning, and this result has me a little cautious. How do we improve a model to handle abnormalities like this?

To be fair, I did upload a pic of my cat and it registered as 99.99% certain, so that was nice.

**Comment by tcapelle** [comment]
your model has only to possible outputs, cat or dog, you could train a model with three labels:
- cat
- dog
- other
Also, the problem you see, is that you get a logits that will forcefully choose between these two, this function only looks at the difference on "utility" (in economics terms) and will prefer the one with high value. Probably both are low, but cat is higher, hence the output.

**Comment by ghosh-r** [comment]
You should use the [forums](https://forums.fast.ai) for this kind of thing.
